met-cron/cron/crontab

```diff
@@ -6,4 +6,6 @@
 */5 * * * * default cd /met-cron && ./run_met_publish.sh
 # PURGE Runs At every 15 days.
 0 0 * * 0 default cd /met-cron && ./run_met_purge.sh
+# REDACT COMMENTS Runs At every 14 days.
+0 0 */15 * * default cd /met-cron && ./run_met_comment_redact.sh
```

Comment:
Okay noted, the logic now is actually "in the last 14 days" rather than "for 14 days". Any ideas how we can limit the query for closed in the last 14 days. Because we don't want older already processed ones right?

Reply:
yeah, i'm thinking we might need some other flag to tell when the comment was redacted!?

Reply:
As discussed, leaving it to fetch everything until we resolve it with @saravanpa-aot